backend/complexity_analyser/src/main/scala/complexity_analyser/HaskellProcessor.scala

```diff
@@ -180,27 +177,22 @@ class HaskellProcessor(modelAnswer: File, studentAnswer: File) {
   def calculateScore(deltas: ArrayBuffer[(String, Double)]) = {
     var score = 100
     val annotations = new ArrayBuffer[Error]
-    for ((n, v) <- deltas) {
-      val diff = Math.abs(v).round
-      if (diff > 50) {
-        if (score > 0) {
-          score -= (diff / 8).toInt
-        }
+    var eff = ""
+    for ((n, diff) <- deltas) {
+      if (Math.abs(diff) > 25000) {
+        score -= (diff / 20000).toInt
```

Comment:
yeah, me and dom said we're gonna comment a bit on the numbers and stuff. it is a constant threshold indeed though.